docker/test-server/Dockerfile

```diff
@@ -21,18 +21,19 @@ RUN set -ex ;\
     cd /app ;\
     mkdir -p /etc/etebase-server ;\
     cp docker/test-server/etebase-server.ini /etc/etebase-server ;\
+    cp docker/test-server/etebase-server.ini /app ;\
     sed -e '/ETEBASE_CREATE_USER_FUNC/ s/^#*/#/' -i /app/etebase_server/settings.py ;\
     chmod +x docker/test-server/entrypoint.sh
 
 # this is a test image and should start up quickly, so it starts with the DB
 # and static data already fully set up.
 RUN set -ex ;\
-    cd /app ;\
-    python manage.py migrate ;\
-    python manage.py collectstatic --noinput
+    cd /app # ;\
+    # python manage.py migrate ;\
+    # python manage.py collectstatic --noinput
 
 ENV ETESYNC_VERSION=${ETESYNC_VERSION}
-VOLUME /data
+#VOLUME /data
```

Comment:
Why comment out the volume? It needs to store files. In prod we will use s3, but in local we would use local volume.

Reply:
Volume line is moved to docker-compose file, if we use docker-compose we mount volume through docker-compose file.